packages/flutter_tools/lib/runner.dart

```diff
@@ -63,31 +63,55 @@ Future<int> run(
     StackTrace? firstStackTrace;
     return runZoned<Future<int>>(() async {
       try {
+        if (args.contains('--disable-telemetry') &&
+            args.contains('--enable-telemetry')) {
+          throwToolExit(
+              'Both enable and disable telemetry commands were detected '
+              'when only one can be supplied per invocation.',
+              exitCode: 1);
+        }
```

Comment:
Is an exit code of `1` appropriate for this exit code? @gspencergoog

Reply:
Yes, I think so.  Anything non-zero would work.